.github/workflows/checks.yml

```diff
@@ -73,8 +73,36 @@ jobs:
       - name: Test
         run: npm run test
 
+  e2e:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 60
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Cache and restore samples
+        uses: actions/cache@v2
+        id: samples
+        with:
+          path: ./samples/
+          key: ${{ runner.os }}-samples-${{ hashFiles('**/scripts/download-samples.sh') }}
+          restore-keys: |
+            ${{ runner.os }}-samples-
+      - name: Install dependencies
+        run: npm ci
+      - name: Install Playwright Browsers
+        run: npx playwright install --with-deps chromium
```

Comment:
It seems if you run with Google Chrome instead of Chromium, you don't need this step (which is quite slow).
See https://github.com/dfinity/nns-dapp/pull/2225/commits/ae153b1a1663605f14374b0dcd02bdc1d21a6cae

Reply:
Cool, thanks. Done.

Notes: I regenerated all screenshots, darwin and linux